llama-index-core/llama_index/core/base/llms/types.py

```diff
@@ -13,7 +13,7 @@
     Optional,
     Union,
 )
-
+from pydantic import FilePath, field_validator
```

Comment:
actually quick nite, I would prefer `from llama_index.core.bridge.pydantic import ....` instead of directly from pydantic

Reply:
Good catch, I did that because one of the two wasn't immediately available in the bridge, I'll fix that.

Side question: do we still need the bridge now that we migrated to v2?

Reply:
I think the bridge is helpful for future breakage 😅 our bridge imports won't change, but pydantic still isn't the most stable